src/os/bluestore/KernelDevice.cc

```diff
@@ -24,15 +24,25 @@
 #include "include/stringify.h"
 #include "common/errno.h"
 #include "common/debug.h"
-#include "common/blkdev.h"
 #include "common/align.h"
 #include "common/blkdev.h"
+#include "common/perf_counters.h"
 
 #define dout_context cct
 #define dout_subsys ceph_subsys_bdev
 #undef dout_prefix
 #define dout_prefix *_dout << "bdev(" << this << " " << path << ") "
 
+enum {
+  l_bluestore_kerneldevice_first = 633430,
+  l_bluestore_kerneldevice_aio_write_lat,
```

Comment:
@liewegas . As a question: why dose it set first a unique value rather than default zero?  By PerfCounterBuilder' name, we can distinguish .

Reply:
This is a kludgey "type safety" type of check that prevents you from accidentally passing l_osd_foo instead of l_bluestore_foo by verifying the constant is in the right range.